<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diagnosticwindow</class>
 <widget class="QDialog" name="diagnosticwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Диагностика </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fnameEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Иванов Иван Иванович</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telnumberEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>89998882233</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="carEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Toyouta</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Телефон</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Машина</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>добавить</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="kuzovButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Кузовные работы</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="paintButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Покраска</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="washButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Техническая мойка</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="balancButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Балансировка колёс</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="replaceButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Замена шин</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО Мастера</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fmasternameEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Иванов Иван Иванович</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="defsysButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Установка систем безопасности</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="replacediskButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Замена дисков</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
